docs/Displaying a User Interface/Scrollable Message/index.md

```diff
@@ -12,7 +12,7 @@ The message will persist on the screen until the timeout has elapsed or the user
 Currently, you can only create a scrollable message view to display on the screen using RPCs.
 
 !!! NOTE
-The @![iOS]`SDLScreenManager`!@ @![android, javaSE, javaEE, javascript]`ScreenManager`!@ takes soft button ids 0 - 10000. Ensure that if you use custom RPCs, that the soft button ids you use are outside of this range.
+The @![iOS]`SDLScreenManager`!@ @![android, javaSE, javaEE, javascript]`ScreenManager`!@ uses soft button ids 0 – 10000. Ensure that if you use custom RPCs—such as this one—that the soft button ids you use are outside of this range (i.e. > 10000).
```

Comment:
Should we add a similar note to the Alerts/Subtle Alerts page explaining why the examples there use 10001 as the set Id?

Reply:
I missed the note, this is already in the Alert/SubtleAlert page